templates/addons/aws/modules/vpc/main.tf

```diff
@@ -15,3 +15,22 @@ module "vpc" {
   one_nat_gateway_per_az = false
   enable_dns_hostnames   = true
 }
+
+data "aws_route_tables" "private_route_table" {
+  vpc_id = module.vpc.vpc_id
+
+  filter {
+    name   = "tag:Name"
+    values = ["${var.env_namespace}-vpc-private"]
+  }
+}
+
+resource "aws_vpc_endpoint" "s3" {
+  vpc_id          = module.vpc.vpc_id
+  service_name    = "com.amazonaws.${var.region}.s3"
```

Comment:
We haven't defined the `region` variable for this module, so this line can raise error 🙏

Reply:
e00fd45 🙈  Nice catch, I forgot about that one 🙇